src/main/java/com/github/osisoft/ocs_sample_library_preview/sds/TypesClient.java

```diff
@@ -89,7 +91,7 @@ public String createType(String tenantId, String namespaceId, SdsType typeDef) t
     }
 
     /**
-     * get the type
+     * get stream type
```

Comment:
i think all of the type calls don't work on the community and in this case i think the comment is ok

Reply:
Correct, I reset this file.